changelogs/fragments/deprecator.yml

```diff
@@ -0,0 +1,17 @@
+minor_changes:
+  - modules - The ``AnsibleModule.deprecate`` function no longer sends deprecation messages to the target host's logging system.
+  - ansible-test - Improved ``pylint`` checks for Ansible-specific deprecation functions.
+  - deprecations - Removed support for specifying deprecation dates as a ``datetime.date``, which was included in an earlier 2.19 pre-release.
```

Comment:
instead of having 2 clog entries for things that were added and then removed but nto released, just remove the p rev entry. This will be very confusing to those that read through clog if they don't read the full thing and/or remember all previous entries.